From: Ben Hutchings <ben@decadent.org.uk>
Date: Sat, 1 Jan 2022 14:59:07 +0100
Subject: [klibc] ia64: Fix out-of-tree build
Origin: https://git.kernel.org/pub/scm/libs/klibc/klibc.git/commit/?id=2114e20cf16bbfb45a94aaf978d62d08d8968414

The linker script is currently not found in an out-of-tree build.
The filename needs to start with $(srctree) to make this work.

Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
 usr/klibc/arch/ia64/MCONFIG | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usr/klibc/arch/ia64/MCONFIG b/usr/klibc/arch/ia64/MCONFIG
index f2a25395..db6a4cee 100644
--- a/usr/klibc/arch/ia64/MCONFIG
+++ b/usr/klibc/arch/ia64/MCONFIG
@@ -11,7 +11,7 @@ KLIBCARCHREQFLAGS = -mno-pic
 KLIBCOPTFLAGS += -Os
 KLIBCBITSIZE  = 64
 
-KLIBCSHAREDFLAGS = -T $(src)/arch/$(KLIBCARCH)/klibc.ld
+KLIBCSHAREDFLAGS = -T $(srctree)/$(src)/arch/$(KLIBCARCH)/klibc.ld
 
 # Kernel has never used stack trampolines
 KLIBCEXECSTACK := n
